src/Components/fe-lang-toggle/FeLangToggle.js

```diff
@@ -0,0 +1,87 @@
+import { LitElement, html, css } from 'lit-element';
+import { localize, LocalizeMixin } from '@lion/localize';
+import '@lion/button';
+
+export class FeLangToggle extends LocalizeMixin(LitElement) {
+  static get localizeNamespaces() {
+    return [
+      {
+        'fe-lang-toggle': locale =>
+          import(`./stories/translations/${locale}.js`),
+      },
+      ...super.localizeNamespaces,
+    ];
+  }
+
+  firstUpdated() {
+    super.firstUpdated();
+    if (localize.locale === 'en-GB') {
+      this.shadowRoot.querySelector('#en-GB').classList.add('selected');
+    } else {
+      this.shadowRoot.querySelector('#nl-NL').classList.add('selected');
+    }
+  }
+
+  static get properties() {
+    return {};
+  }
+
+  toggleLanguage(locale) {
+    localize.locale = locale;
+    const buttons = this.shadowRoot.querySelectorAll('lion-button');
+    buttons.forEach(button => {
+      button.classList.remove('selected');
+    });
+    buttons.forEach(button => {
```

Comment:
we dont need forEach for adding because your adding to one class only and already click event is added why it needed again ?

Reply:
okay. I will remove the forEach for adding the style